charts/platform-tenant/templates/cluster-policy.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.components.kyverno.enabled -}}
+  {{- if .Values.crdCheck }}
+    {{- include "require.api.kyerno.clusterPolicy" . -}}
+  {{- end }}
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: {{ list .Values.components.kyverno.name $ | include "platform-tenant.format.name.shared" }}
```

Comment:
Names across resources have multiple ways to name them. I feel like we need a standard